.goreleaser.yml

```diff
@@ -189,3 +189,31 @@ blobs:
     region: us-west-2
     folder: "{{ .Env.S3FOLDER }}/archives/{{ .Version }}"
     disable: "{{ .Env.DRY_RUN }}" 
+
+dockers:
+  - ids:
+      - confluent-alpine-amd64
+    image_templates:
+      - "confluentinc/confluent-cli:{{ .Version }}-amd64"
+      - "confluentinc/confluent-cli:latest-amd64"
+    skip_push: "{{ .Env.DRY_RUN }}"
+    dockerfile: "dockerfiles/Dockerfile"
+  - ids:
+      - confluent-alpine-arm64
+    image_templates:
+      - "confluentinc/confluent-cli:{{ .Version }}-arm64"
+      - "confluentinc/confluent-cli:latest-arm64"
+    skip_push: "{{ .Env.DRY_RUN }}"
+    dockerfile: "dockerfiles/Dockerfile"
+
+docker_manifests:
+  - name_template: "confluentinc/confluent-cli:{{ .Version }}"
+    image_templates:
+      - "confluentinc/confluent-cli:{{ .Version }}-amd64"
+      - "confluentinc/confluent-cli:{{ .Version }}-arm64"
+    skip_push: "{{ .Env.DRY_RUN }}"
+  - name_template: "confluentinc/confluent-cli:latest"
```

Comment:
are we pushing 2 images tagged `latest-arm64` and `latest-amd64`? What will happen if they pull `latest`